TestRun/README.md

```diff
@@ -0,0 +1,7 @@
+#TestRun
```

Comment:
This all looks good. Would you mind adding an actual description and ideas etc before merging